src/server/handlers/launchpad.js

```diff
@@ -480,16 +480,9 @@ const internalFindSnaps = async (owner, token) => {
     }
     return snaps;
   } catch (error) {
-    // At least for the moment, we just wrap the error we get from
-    // Launchpad.
-    const text = await error.response.text();
```

Comment:
We were assuming error will have a `response` object (as any error from LP client would), but if error was not coming from LP client and it didn't have `response` property that would throw here again with `Cannot read property 'text' of undefined` that would be then sent in API response hiding the real error behind it.